nexus/db-model/src/inventory.rs

```diff
@@ -1278,21 +1278,9 @@ impl InvOmicronZone {
                 inv_omicron_zone.dns_gz_address_index =
                     Some(SqlU32::from(*gz_address_index));
             }
-            OmicronZoneType::InternalNtp {
-                address,
-                ntp_servers,
-                dns_servers,
-                domain,
-            } => {
+            OmicronZoneType::InternalNtp { address } => {
                 // Set the common fields
                 inv_omicron_zone.set_primary_service_ip_and_port(address);
-
-                // Set the zone specific fields
-                inv_omicron_zone.ntp_ntp_servers = Some(ntp_servers.clone());
-                inv_omicron_zone.ntp_dns_servers = Some(
-                    dns_servers.iter().cloned().map(IpNetwork::from).collect(),
-                );
-                inv_omicron_zone.ntp_domain.clone_from(domain);
```

Comment:
I was expecting a change to the db model types and database schema and then I thought "this is gonna be complicated" because we might have old inventories that have this information.  But I guess maybe: no change is strictly necessary because these are NULL-able fields anyway, so they will just be NULL for all new inventories?

Should we still drop the columns?  This technically loses data from old inventories where they were populated but I think we're saying this data is useless anyway.

Reply:
We cannot drop the columns; they were used (i.e., set to non-NULL values) by both internal NTP and boundary NTP; they are still used by boundary NTP. Good question though - I need to update some comments that refer to them as applying to both kinds.

Reply:
Updated the comments on these columns in `dbinit.sql` in 48ee15bad